### PR TITLE
Refactor shell scripts in order to make producing different outputs smoother

### PR DESCRIPTION
**Pull request by jarkkoka**
This change is [Reviewable](https://reviewable.io/reviews/hsldevcom/jore4-digiroad-import-experiment/7)

### CODE REVIEW

**Review by pallost — CHANGES_REQUESTED** [review]
This is a good first implementation of the tool. However with having so many docker run, docker kill commands, it's really hard to follow what's actually going on. It would be much nicer if this all was contained in a docker-compose file and the actual commands were called with "docker-compose exec" (https://docs.docker.com/compose/reference/exec/)

> *[Reviewable](https://reviewable.io/reviews/hsldevcom/jore4-digiroad-import-experiment/7#-:-Mi_R1KK2p_VLtfEbqgk:b-57jn75)* status: 0 of 6 files reviewed, 3 unresolved discussions (waiting on @jarkkoka)

___
*[export_pgdump.sh, line 18 at r1](https://reviewable.io/reviews/hsldevcom/jore4-digiroad-import-experiment/7#-Mi_QTfY9O3KNF5g2IZj:-Mi_QTfY9O3KNF5g2IZk:bq6zklj) ([raw file](https://github.com/hsldevcom/jore4-digiroad-import-experiment/blob/f732893e6176519d3d8d5ec36c36afef9a2d0a25/export_pgdump.sh#L18)):*
> ```Shell
> 
> # Export pg_dump file.
> docker run -it --rm --link "${DOCKER_CONTAINER}":postgres -v ${WORK_DIR}/pgdump/:/tmp/pgdump $DOCKER_IMAGE \
> ```

should run `mkdir -p ./pgdump` before this to make sure that the directory is created in your own name. Otherwise if docker creates it when the volume is mapped, it will be owner by root
___
*[README.md, line 7 at r1](https://reviewable.io/reviews/hsldevcom/jore4-digiroad-import-experiment/7#-Mi_PXTnECTzCDBQ92mw:-Mi_PXTnECTzCDBQ92mx:b-ycay1u) ([raw file](https://github.com/hsldevcom/jore4-digiroad-import-experiment/blob/f732893e6176519d3d8d5ec36c36afef9a2d0a25/README.md#L7)):*
> ```Markdown
> This repository provides scripts to download Digiroad shapefiles for road geometries and makes transformations required by JORE4 components/services.
> 
> Firstly, Digiroad links and other related information is downloaded and imported from shapefiles into a PostGIS database contained in a Docker container. The data is then processed in the database. After processing, the data can be exported in a couple of formats relevant to JORE4 services.
> ```

could maybe somehow mention here that the PostGIS database is only for local and temporary use while the tool is running, there's no need to persist it anywhere
___
*[set_env_vars.sh, line 10 at r1](https://reviewable.io/reviews/hsldevcom/jore4-digiroad-import-experiment/7#-Mi_PzvN-_mwwcsp_y2L:-Mi_PzvO-_tdD_fBuV9r:b-57dgpc) ([raw file](https://github.com/hsldevcom/jore4-digiroad-import-experiment/blob/f732893e6176519d3d8d5ec36c36afef9a2d0a25/set_env_vars.sh#L10)):*
> ```Shell
> 
> export DOCKER_IMAGE="jore4/postgis-digiroad"
> export DOCKER_CONTAINER="jore4-postgis-digiroad"
> ```

these could be named rather something like "DIGIROAD_IMPORTER_IMAGE" and "DIGIROAD_IMPORTER_CONTAINER" to clarify what docker image and container we are talking about

**Review by jarkkoka — COMMENTED** [review]
I understand and agree. However, refactoring the script code into a docker-compose form would be such a tedious task (not least because of upcoming commits and resulting merge conflicts) that I would postpone that change to a later stage when functioning is otherwise more ready.

> *[Reviewable](https://reviewable.io/reviews/hsldevcom/jore4-digiroad-import-experiment/7#-:-MjdRQKs4m9DEOCq_81S:b-t6anbl)* status: 0 of 6 files reviewed, 3 unresolved discussions (waiting on @pallost)

___
*[export_pgdump.sh, line 18 at r1](https://reviewable.io/reviews/hsldevcom/jore4-digiroad-import-experiment/7#-Mi_QTfY9O3KNF5g2IZj:-MjdOS9P6hNlSiU8IMZD:b-1lfvng) ([raw file](https://github.com/hsldevcom/jore4-digiroad-import-experiment/blob/f732893e6176519d3d8d5ec36c36afef9a2d0a25/export_pgdump.sh#L18)):*
<details><summary><i>Previously, pallost (Tamas Pallos) wrote…</i></summary><blockquote>

should run `mkdir -p ./pgdump` before this to make sure that the directory is created in your own name. Otherwise if docker creates it when the volume is mapped, it will be owner by root
</blockquote></details>

Will do this.
___
*[README.md, line 7 at r1](https://reviewable.io/reviews/hsldevcom/jore4-digiroad-import-experiment/7#-Mi_PXTnECTzCDBQ92mw:-MjdRAOPCVvpEJZiFL5j:b-1lfvng) ([raw file](https://github.com/hsldevcom/jore4-digiroad-import-experiment/blob/f732893e6176519d3d8d5ec36c36afef9a2d0a25/README.md#L7)):*
<details><summary><i>Previously, pallost (Tamas Pallos) wrote…</i></summary><blockquote>

could maybe somehow mention here that the PostGIS database is only for local and temporary use while the tool is running, there's no need to persist it anywhere
</blockquote></details>

Will do this.
___
*[set_env_vars.sh, line 10 at r1](https://reviewable.io/reviews/hsldevcom/jore4-digiroad-import-experiment/7#-Mi_PzvN-_mwwcsp_y2L:-MjdSXjSE35KU39fpa7O:b-4iym97) ([raw file](https://github.com/hsldevcom/jore4-digiroad-import-experiment/blob/f732893e6176519d3d8d5ec36c36afef9a2d0a25/set_env_vars.sh#L10)):*
<details><summary><i>Previously, pallost (Tamas Pallos) wrote…</i></summary><blockquote>

these could be named rather something like "DIGIROAD_IMPORTER_IMAGE" and "DIGIROAD_IMPORTER_CONTAINER" to clarify what docker image and container we are talking about
</blockquote></details>

Is it ok to do this later (after a few of the currently done commits in the queue are reviewed)? I mean changing this would result in quite many merge conflicts. There are already four or five different branches to be maintained (two for both R and K material).

**Review by pallost — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/hsldevcom/jore4-digiroad-import-experiment/7)* status: 0 of 6 files reviewed, 1 unresolved discussion (waiting on @jarkkoka)

___
*[set_env_vars.sh, line 10 at r1](https://reviewable.io/reviews/hsldevcom/jore4-digiroad-import-experiment/7#-Mi_PzvN-_mwwcsp_y2L:-MjhRl83BOYboZ-Mu2j_:b-d0r6vz) ([raw file](https://github.com/hsldevcom/jore4-digiroad-import-experiment/blob/f732893e6176519d3d8d5ec36c36afef9a2d0a25/set_env_vars.sh#L10)):*
<details><summary><i>Previously, jarkkoka (Jarkko Kaura) wrote…</i></summary><blockquote>

Is it ok to do this later (after a few of the currently done commits in the queue are reviewed)? I mean changing this would result in quite many merge conflicts. There are already four or five different branches to be maintained (two for both R and K material).
</blockquote></details>

yeah, this is not essential
